_includes/manuals/1.12/3.1.3_puppet_versions.md

```diff
@@ -86,7 +86,7 @@ Puppet Server is the application for serving Puppet agents used by default in Pu
   <tr>
     <td>2.0 - 2.1</td>
     <td>Limited support</td>
-    <td>Requires server_use_legacy_use_conf, server_puppetserver_version to be set</td>
+    <td>Requires server_use_legacy_host_conf, server_puppetserver_version to be set</td>
```

Comment:
It's actually `server_use_legacy_auth_conf`

Reply:
oh nevermind, i realized this is actually a setting on the puppetserver.